src/components/form/DateField.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import LuxonUtils from '@date-io/luxon';
+import { LocalizationProvider } from '@material-ui/pickers';
+import { action } from '@storybook/addon-actions';
+import { boolean, select, text } from '@storybook/addon-knobs';
+import React from 'react';
+import { Form } from 'react-final-form';
+import { DateField as DF } from './DateField';
+
+export default { title: 'Components/DateField' };
+
+export const DateField = () => (
+  <LocalizationProvider dateAdapter={LuxonUtils}>
+    <Form onSubmit={action('onSubmit')}>
+      {({ handleSubmit }) => (
+        <form onSubmit={handleSubmit}>
+          <DF
+            name="DateField"
```

Comment:
`name` is the field name so it should always be _camelCase_. Ah appropriate name would be nice too, like `startDate` here.